lang/gtext_jpn.po

```diff
@@ -5363,207 +5363,207 @@ msgstr ""
 msgctxt "Easter egg"
 msgid "The fools who enter your domain, Will never see the light again."
-msgstr "さあ行け！ アバタールはおらぬ。"
+msgstr "愚かにもお前の領地に入った者は、二度と光を見ぬであろう。"
 
 #: guitext:819
 msgctxt "Easter egg"
 msgid "Pain and anguish are your tools, To use upon intruding fools."
-msgstr "ロードは恐怖におののいているぞ。"
+msgstr "苦痛と苦悩がお前の道具。愚かな侵入者に使うがよい。"
 
 #: guitext:820
 msgctxt "Easter egg"
 msgid ""
 "When good guys head right for your treasure, Hunt and kill them at your "
 "leisure."
-msgstr "侵入者どもは滅びるであろう、お前のダンジョンの拷問室で。"
+msgstr "善人がお前の宝を狙ったら、楽しみながら狩り立て殺せ。"
 
 #: guitext:821
 msgctxt "Easter egg"
 msgid ""
 "Protect your gold and Dungeon Heart, And slap your minions till they fart."
-msgstr "たとえ心は純粋でも、侵入者どもは息絶える、"
+msgstr "財宝とダンジョンハートを守り抜け。そして手下に活を入れよ、奴らが思わず屁をこくまで。"
 
 #: guitext:822
 msgctxt "Easter egg"
 msgid "Use your creatures' fear of you. To make them do what you want them to."
-msgstr "お前のダンジョンの土の上で。"
+msgstr "お前への手下の恐怖を利用せよ。望み通りに奴らを動かせ。"
 
 #: guitext:823
 msgctxt "Easter egg"
 msgid "Wizard in your dungeon grim? A trap will make short work of him."
-msgstr "お前のダンジョンは暗く冷たく、老ナイトのいるような"
+msgstr "手下のウィザードは不愉快か？ 罠で手早く片付けてしまえ。"
 
 #: guitext:824
 msgctxt "Easter egg"
 msgid "When your creatures give you hell, Stick them in your Prison Cell."
-msgstr "場所ではない。いかに大胆不敵を自称していようとも。"
+msgstr "手下が悩みの種ならば、奴らを牢に閉じ込めろ。"
 
 #: guitext:825
 msgctxt "Easter egg"
 msgid "Torture is the crowning glory, Of your Dungeon Keeper story."
-msgstr "愚かにもお前の領地に入った者は、二度と光を見ぬであろう。"
+msgstr "拷問は、ダンジョンキーパー物語における無上の栄誉なり"
 
 #: guitext:826
 msgctxt "Easter egg"
 msgid "You are the Keeper, chosen one, And being evil is great fun."
-msgstr "苦痛と苦悩がお前の道具。愚かな侵入者に使うがよい。"
+msgstr "お前こそが選ばれしキーパー。悪の道は大いなる楽しみ。"
 
 #: guitext:827
 msgctxt "Easter egg"
 msgid "When your troops are hunger-stricken, Your Hatchery has yummy chicken."
-msgstr "善人がお前の宝を狙ったら、楽しみながら狩り立て殺せ。"
+msgstr "軍隊がハンガーストライキをする時は、ニワトリ小屋があふれる。"
 
 #: guitext:828
 msgctxt "Easter egg"
 msgid "Fill the Avatar with dread. Really make him wet the bed."
-msgstr "財宝とダンジョンハートを守り抜け。そして手下に活を入れよ、"
+msgstr "アバタールの心を恐怖で満たしてやれ。思わずズボンを濡らすほど。"
 
 #: guitext:829
 msgctxt "In-game hint message"
 msgid ""
 "Imps are the lifeblood of your dungeon. Like blood, they must be allowed to "
 "circulate."
-msgstr "奴らが思わず屁をこくまで。"
+msgstr "インプはお前のダンジョンの命の源なり。それは血のごとく、循環しているに違いな"
+"い。"
 
 #: guitext:830
 msgctxt "In-game hint message"
 msgid ""
 "Feeling mean? Set your minions to imprison and when your cells are full, "
 "spend some time torturing..."
-msgstr "お前への手下の恐怖を利用せよ。望み通りに奴らを動かせ。"
+msgstr ""
 
 #: guitext:831
 msgctxt "In-game hint message"
 msgid ""
 "If your creatures get unhappy, slap some sense into them. They'll soon "
 "realise how lucky they are."
-msgstr "手下のウィザードは不愉快か？ 罠で手早く片付けてしまえ。"
+msgstr ""
 
 #: guitext:832
 msgctxt "In-game hint message"
 msgid ""
 "The underworld is your domain and the adventurers come to steal it. Never "
 "show pity, even when the intruders are defenceless and pathetic."
-msgstr "手下が悩みの種ならば、奴らを牢に閉じ込めろ。"
+msgstr ""
```

Comment:
And this one?

Reply:
This translation is telling you to imprison your creatures....

Reply:
Are you sure there correct translations aren't available in other places? If they aren't, could you mark all the ones that are now empty/incorrect/partial as '#fuzzy'?